src/scripts/main.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const gallery = [...document.querySelectorAll('.list-item__link')];
+
+gallery.forEach((img, index) => {
```

Comment:
let's change the code to have only one eventListener and get element by using `evt.target`